sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy

```diff
@@ -182,6 +217,34 @@ class BlockBlobAPITest extends APISpec {
         ]
     }
 
+    // Override name to prevent BinaryData.toString() invocation by test framework.
+    @Unroll("#featureName #iterationIndex")
+    def "Stage block min with BinaryData Async"() {
+        setup:
+        def stepVerifier = StepVerifier.create(blockBlobAsyncClient.stageBlockWithResponse(new BlockBlobStageBlockOptions(getBlockID(), binaryData))
+            .flatMap(resp -> {
+                if (resp.getStatusCode() == 201) {
+                    return blockBlobAsyncClient.listBlocks(BlockListType.ALL)
+                    .map(bl -> bl.getUncommittedBlocks().size() == 1)
+                }
+                return Mono.just(true)
+            }))
+
+        expect:
+        stepVerifier.assertNext({
+            assert it
+        }).verifyComplete()
+
+        where:
+        binaryData << [
```

Comment:
Woah, I didn't know this syntax was possible! This is much nicer than the other format single value parameterization